address_book/contacts/forms.py

```diff
@@ -0,0 +1,23 @@
+from django import forms
+from django.core.validators import RegexValidator
+from contacts.models import Contact
+from django.shortcuts import get_object_or_404
+
+class ContactForm(forms.ModelForm):
+    phone_number = forms.CharField(max_length=15, validators=[
+        RegexValidator(r'^\d{1,15}$', message="Phone number must be numeric.")
```

Comment:
Please also allow "+" in the start, update the regex.

Reply:
yes updated